### DISCUSSION TITLE
Execute Tests in Parallel across multiple browsers with run-workers

### DISCUSSION THREAD

**Discussion by karanshah-browserstack** [codeceptjs/CodeceptJS#3596] [General]
Currently, we don’t have a in built support for running the tests across multiple browsers with run-workers setup. We have to create a custom runner and manually listen to codeceptJS events for reporting.
Is this approach the only recommendation from your side? Is it possible to have a built-in capability to do this similar to what we had in run-multiple?

**Comment by adudyak** [comment]
https://codecept.io/parallel/#parallel-execution-by-workers-on-multiple-browsers
